x/zoneconcierge/keeper/ibc_packet_btc_staking_consumer_event.go

```diff
@@ -72,3 +74,61 @@ func (k Keeper) HandleConsumerRegistration(
 
 	return k.btcStkKeeper.RegisterConsumer(ctx, consumerRegisterData)
 }
+
+func (k Keeper) HandleConsumerSlashing(
+	ctx sdk.Context,
+	destinationPort string,
+	destinationChannel string,
+	consumerSlashing *types.ConsumerSlashingIBCPacket,
+) error {
+	clientID, _, err := k.channelKeeper.GetChannelClientState(ctx, destinationPort, destinationChannel)
+	if err != nil {
+		return fmt.Errorf("failed to get client state: %w", err)
+	}
+
+	evidence := consumerSlashing.Evidence
+	if evidence == nil {
+		return fmt.Errorf("consumer slashing evidence is nil")
+	}
```

Comment:
This can be due to a misbehaviour of the consumer side, rather than an issue in Babylon side. So we will need to return an IBC acknowledgement with some error, rather than returning error here.

Similar comments apply to other error cases in this function

Reply:
Ah I see, all good then